ruby/vampires.rb

```diff
@@ -0,0 +1,83 @@
+insurance = ''
+year_of_birth = ''
+age = ''
+age_calc = 0
+age_true = false
+age_false = false
+insurance_yes = false
+insurance_no = false
+like_garlic = false
+hate_garlic = false
+
+puts "How many candidates are there?"
+candidate_number = gets.chomp
+index = 0
+
+while index < candidate_number.to_i
+
+	puts "What is your name?"
+	employee_name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	year_of_birth = gets.chomp.to_i
+	age_calc = 2017 - year_of_birth
+
+	if age == age_calc
+		age_true = true
+	else 
+		age_false = true
+	end
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	garlic_answer = gets.chomp
+
+	if garlic_answer == "y"
+		like_garlic = true
+	elsif garlic_answer == "n"
+		hate_garlic = true
+	else 
+		puts "error."
+	end
+
+	puts "Would you like to enroll in the company’s health insurance? (y/n)"
+	insurance = gets.chomp
+
+	if insurance == "y"
+		insurance_yes = true
+	elsif insurance == "n"
+		insurance_no = true
+	end
+
+	sunshine_allergy = false
+	allergy = ""
+
+	until allergy == "done"
+		puts "Enter any allergies you have one at a time. Type 'done' when you are finished."
+		allergy = gets.chomp
+		if allergy == "sunshine"
+			sunshine_allergy = true
+			allergy = "done"
+		end
+	end
+
+	if sunshine_allergy == true
```

Comment:
If we have a variable that points to a boolean, we don't need to compare it to another boolean; i.e. if true == true, instead we can just say if true like the below

`if sunshine_allergy`